### PR TITLE
Update UG - sikai00

### PR DESCRIPTION
**Pull request by sikai00**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -66,33 +66,38 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
-Shows a message explaning how to access the help page.
+Shows a message explaining how to access the help page.
 
 ![help message](images/helpMessage.png)
 
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding a client: `addClient`
```
Comment (Thing1Thing2) [inline docs/UserGuide.md:76]: Leaving a note here to document contribution: add client section added to UG.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -66,33 +66,38 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
+* `addClient n/John Tan p/0123456789`
+* `addClient n/John Tan p/0123456789 e/johntan@insurec.com`
 
 ### Listing all persons : `list`
 
 Shows a list of all persons in the address book.
 
 Format: `list`
 
+### Listing meetings: `listMeeting`
```
Comment (Thing1Thing2) [inline docs/UserGuide.md:95]: List meetings command section added to UG.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -148,7 +153,7 @@ Clears all entries from the address book.
 
 Format: `clear`
 
-### Exiting the program : `exit`
+### Exiting MyInsuRec : `exit`
```
Comment (Thing1Thing2) [inline docs/UserGuide.md:156]: Exit command section added to UG.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -66,33 +66,38 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
+* A client must have a NAME and a PHONE_NUMBER.
+* Email is optional.
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `addClient n/John Tan p/0123456789`
+* `addClient n/John Tan p/0123456789 e/johntan@insurec.com`
 
 ### Listing all persons : `list`
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:89]: Please help to change all `persons` into `clients` to standardise.

File: docs/UserGuide.md [hunk 5]
```diff
@@ -66,33 +66,38 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 
 Format: `list`
 
+### Listing meetings: `listMeeting`
+
+List all meetings in MyInsuRec.
+
+Format: `listMeeting`
+
 ### Editing a person : `edit`
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:101]: Please help to change all `persons` into `clients` to standardise.

**Review by ThomasHoooo — CHANGES_REQUESTED** [review]

**Review by Thing1Thing2 — APPROVED** [review]
lgtm, thank you for completing the command summary section.

**Pull request merged**